app/models/container_node.rb

```diff
@@ -75,6 +75,12 @@ def ipaddress
   end
 
   def cockpit_url
-    URI::HTTP.build(:host => ipaddress, :port => 9090)
+    if MiqServer.active_miq_servers.where(:has_active_cockpit_ws => true)
```

Comment:
I just don't understand the use of a boolean flag vs. looking for servers with the role active.